tests/deployment/test_client_server.py

```diff
@@ -89,18 +96,31 @@ def test_client_server_sklearn(
         # FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/4014
         pytest.skip("Skipping KNN, because FHE predictions and clear ones are differents.")
 
+    if (
+        get_model_name(model_class) == "SGDClassifier"
+        and parameters.get("n_classes") != 2
+        and getattr(model_class, "keywords", {}).get("fit_encrypted") is True
+    ):
+        # Skipping SGDClassifier for this test
+        # FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/4182
+        pytest.skip("Skipping SGDClassifier, because it only supports 2 classes.")
+
     # Generate random data
     x, y = load_data(model_class, **parameters)
 
     x_train = x[:-1]
     y_train = y[:-1]
     x_test = x[-1:]
 
-    model = instantiate_model_generic(model_class, n_bits=n_bits)
+    # Instantiate the model
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore", category=UserWarning)
```

Comment:
what is this userwarning ?

Reply:
It's this that appear when we use `fit_encrypted = True`

https://github.com/zama-ai/concrete-ml/blob/da83329717df905da6e69e946619339cbd51a5cc/src/concrete/ml/sklearn/linear_model.py#L239-L243

Reply:
got it thanks !